template.yaml

```diff
@@ -443,7 +466,78 @@ Resources:
             Path: /config/positive-outcome
             Method: put
 
+  SetFileURL:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: file.setFileURL
+      Runtime: nodejs12.x
+      CodeUri: src/
+      Policies:
+      - S3WritePolicy:
+          BucketName:
+            Ref: Bucket
+      Environment:
+        Variables:
+          BUCKET_NAME:
+            Ref: Bucket
+      Events:
+        SetFileURL:
+          Type: Api
+          Properties:
+            RestApiId: 
+              Ref: RekommendEditorApi
+            Path: /set-file-url/{extension}
+            Method: get
+
+  GetFileURL:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: file.getFileURL
+      Runtime: nodejs12.x
+      CodeUri: src/
+      Policies:
+      - S3ReadPolicy:
+          BucketName:
+            Ref: Bucket
+      Environment:
+        Variables:
+          BUCKET_NAME:
+            Ref: Bucket
+      Events:
+        GetFileURL:
+          Type: Api
+          Properties:
+            RestApiId: 
+              Ref: RekommendEditorApi
+            Path: /get-file-url/{filename}
```

Comment:
Instead of using this path, it should, I suggest, be something like `/image/{filename}`. This means that when a webpage tries to load /image/youImage.jpg, it will be redirected to the S3 special URL using a 302

Reply:
changed to `image/{filename}`